apps/src/studio/studio.js

```diff
@@ -2977,7 +2977,8 @@ Studio.execute = function () {
           // Expose `Studio.Globals` to success/failure functions. Setter is a no-op.
           Object.defineProperty(Studio, 'Globals', {
```

Comment:
I don't know exactly offhand, but I believe it's used by generated code that runs in our sandbox (i.e. in play lab, the user drags in some blocks, which we turn into JS and execute. Some of that code includes references to Studio.Globals)